src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps2_mqtt_client_module.c

```diff
@@ -76,20 +90,106 @@ ADUC_AGENT_CONTRACT_INFO g_adpsMqttClientContractInfo = {
     "Microsoft", "Azure DPS2 MQTT Client  Module", 1, "Microsoft/AzureDPS2MQTTClientModule:1"
 };
 
+// Forward declarations
```

Comment:
Can it just #include the header file(s) for these? If there are circular references, should the 2 be combined into one module?